ush/python/pygfs/task/land_analysis.py

```diff
@@ -307,6 +313,10 @@ def execute(self) -> None:
         for key in keys:
             localconf[key] = self.task_config[key]
 
+        logger.info("Generating IMS snow OBS")
+        if f"{ self.runtime_config.cyc }" == '18':
+            self.prepare_IMS()
```

Comment:
This is preparing the observation.  IIRC, this was part of the preplandobs job

Reply:
@aerorahul From our email conversations on Nov 30, 2023 as below, we have agreement to bring this IMS snow processing into `landanl` job. 

```
Jiarui/Mike,

Rahul and I just had a quick chat, and he is fine with combining the obs preprocessing with the OI job, 
especially since we will have 1 high res OI job and a set of meta tasks for multiple ensemble OI jobs.

Jiarui, please proceed and let me know if you have any questions or how I can help.

-Cory
```

Reply:
yes. That is true and is fine.
However, this is not complete and I don't think is necessary for the snowDA test to work.
Besides, if this is going to be added here, then the counterpart in [exglobal_prep_land_obs.py](https://github.com/NOAA-EMC/global-workflow/blob/7759163c668eb6ccc11dc1ecd39c0dc5d433cdc1/scripts/exglobal_prep_land_obs.py#L24) should be removed.

Moving of `prepare_IMS` from the prep job to the inline analysis job should be its own issue and PR after the test is added.